autogen/agentchat/groupchat.py

```diff
@@ -15,6 +15,7 @@
 from ..runtime_logging import log_new_agent, logging_enabled
 from .agent import Agent
 from .chat import ChatResult
+from .contrib.capabilities import transform_messages
```

Comment:
Thank you for the PR, @marklysze ! Could we add a try...except... to the import? As group chat is in the core while transform_messages is in the contrib, it would be better that core features can still work if something in contrib changed.

Reply:
Hey @thinkall, thanks for suggestion and makes sense.

Do you know of any example in the codebase that has this? I've had a search through as I wanted to try and follow a previous example.

Otherwise, do you envisage something like this:
```
try:
    from .contrib.capabilities import transform_messages
except ImportError:
    transform_messages = None
```

Then perform transforms only `if transform_messages is not None` and show a `logger.warning` if they set the `select_speaker_transform_messages` value.

Reply:
Hi @marklysze , I think there is no previous example. And yes, this is what in my mind.

Reply:
Hey @thinkall, I've updated the code to handle the non-core module and put out a warning if the value is set. Can you let me know if that's good to go?